content/en/containers/amazon_ecs/_index.md

```diff
@@ -29,11 +29,21 @@ algolia:
 
 Amazon ECS is a scalable, high-performance container orchestration service that supports Docker containers. With the Datadog Agent, you can monitor ECS containers and tasks on every EC2 instance in your cluster.
 
+To configure your ECS with Datadog, you can either use **Fleet Automation** or **install manually**. If you prefer to install manually, run one Agent container per EC2 host by creating a Datadog Agent task definition and deploying it as a Daemon service. Each Agent then monitors the other containers on its host. See the [Install manually](#install-manually) section for more details.
+
+
+## Setup with Fleet Automation
```

Comment:
This should either be "Set up with Fleet Automation" or "Fleet Automation setup"